code/src/constants/routes.ts

```diff
@@ -0,0 +1,14 @@
+const routes = {
+  home: {
+    path: "/"
+  },
+  products: {
+    path: "/products",
+    route: "products"
```

Comment:
Why you are using route property. What's different between path. Do we really need it?

Reply:
Well, in this case both properties can be used in the same way, so `route` can be removed. But in some cases, we need absolute path (`path` property) as well as relative path (relative to parent's path, `route` property). `Path` we can use in links for example. `route` property we can use inside of our router, where each route can accept children whose paths must be relative to their parent's path. Maybe this needs better naming, cause just `route` and `path` are not very descriptive.